src/BDCS/API/V0.hs

```diff
@@ -55,7 +55,7 @@ import           BDCS.DB
 import           BDCS.Depclose(depcloseNames)
 import           BDCS.Depsolve(formulaToCNF, solveCNF)
 import           BDCS.Groups(groupIdToNevra)
-import           BDCS.Projects(findProject, getProject)
+import           BDCS.Projects(findProject, getProject, allProjects)
```

Comment:
See the comment I made elsewhere about naming that function `projects`.  Also you'll want to update the cabal file to require at least bdcs 0.1.1.  And then I will need to do a new release.